src/components/renderer/Kopytko.brs

```diff
@@ -2,6 +2,7 @@ sub init()
   m.state = {}
   m.elementToFocus = Invalid
 
+  m._enabledErrorHandling = Type(componentDidCatch) <> "<uninitialized>"
```

Comment:
```suggestion
  m._enabledErrorCatching = Type(componentDidCatch) <> "<uninitialized>"
```
What do you think?

Without bs_const I won't be able to catch original error (during development for example). Working with supressed or rethrown errors might be a nightmare

Reply:
I prefer Handling, but it doesn't matter to me so much, I can change to Catching.

But if someone is suppressing the error for some reason, it shouldn't crash.

Some app components could always extend KopytkoGroup and have any logic the dev wants, it could be embedded there, so for example it could throw an error if they want.
And use this component instead of the KopytkoGroup.

Reply:
@tomek-r

For example, with current solution, if you want to handle error but have it thrown for dev environment, you can create your component that will extend KopytkoGroup

```xml
<?xml version="1.0" encoding="utf-8" ?>

<component name="MyAppGroup" extends="KopytkoGroup">
  <script type="text/brightscript" uri="MyAppGroup.brs" />
</component>
```

```brightscript
sub componentDidCatch(error as Object, info as Object)
  #if isDevelopmentEnvironment
    ?info
    throw error
  #end if

  ' ... normal production code
end sub
```

Of course, it won't be the ideal solution as the backtrace of the crash won't be full,
but you still have the original error message and the original line that crashed.
The app is stopped also in the component scope (8085 port), so maybe you are not exactly at the line of the function that crashed, but you can check the component context and variables.

It is a trade-off solution I won't argue with that, but it is also an optional one, I don't want to force anybody to use it.

Maybe I will have some idea in the future how it could be improved but, don't have it now.